Microsoft.Toolkit.Uwp.SampleApp/Styles/Generic.xaml

```diff
@@ -1282,7 +1303,25 @@
                                             Grid.Row="4" />
 
                                         <!-- Left nav list -->
-                                        <winui:NavigationViewList x:Name="MenuItemsHost" Grid.Row="6" SingleSelectionFollowsFocus="False" Margin="0,0,0,20" SelectionMode="Single" IsItemClickEnabled="True" HorizontalAlignment="Stretch" SelectedItem="{TemplateBinding SelectedItem}" ItemTemplate="{TemplateBinding MenuItemTemplate}" ItemTemplateSelector="{TemplateBinding MenuItemTemplateSelector}" ItemContainerStyle="{TemplateBinding MenuItemContainerStyle}" ItemContainerStyleSelector="{TemplateBinding MenuItemContainerStyleSelector}" />
+                                        <winui:ItemsRepeaterScrollHost
```

Comment:
Why do we need this scroll host? What does it do? Do the Layout samples need this as well?

Reply:
Good point, safe to add it. It's a shim for working down level ItemsRepeater before 1809: https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.itemsrepeaterscrollhost?view=winui-2.4

Reply:
I'll add this to the samples as part of this PR quick. Somehow the StaggeredLayout bind file got removed from the csproj in a merge elsewhere.